client/batch_sql_translator.py

```diff
@@ -54,11 +55,11 @@ def start_translation(self):
         local_input_dir = self.config.input_directory
         local_output_dir = self.config.output_directory
         tmp_dir = join(dirname(self.config.input_directory), self.__TMP_DIR_NAME)
-        if self.config.macro_maps:
+        if self.macros_processor.enable_macro_substitution:
```

Comment:
Can we avoid having this boolean in the macro processor and just use 'is not None' ?

Reply:
Done.